src/nwb_conversion_tools/datainterfaces/ecephys/ced/ceddatainterface.py

```diff
@@ -37,6 +39,9 @@ def get_all_channels_info(cls, file_path: FilePathType):
         assert HAVE_SONPY, INSTALL_MESSAGE
         return cls.RX.get_all_channels_info(file_path=file_path)
 
-    def __init__(self, file_path: FilePathType, smrx_channel_ids: list, verbose: bool = True):
+    def __init__(self, file_path: FilePathType, verbose: bool = True):
```

Comment:
@h-mayorquin why did you remove this optional arg? iirc this was needed in a prior conversion

Reply:
The old data share no longer exists (wasn't copied to the LBNL drive but instead was a link to their share, which has since been deleted) but I have a hard copy that I'm sending now.

Reply:
@h-mayorquin Confirmed that the previous `.smrx` files can be loaded in the new SpikeInterface without needing the extra argument (or class method)

Reply:
Can we please look into recreating this feature in the new pipeline?

Reply:
Selecting only the channels that you want to load?

Reply:
getting the names of the channels as a class method and selecting only the channels you want to load as the init

Reply:
Ok, check:

https://github.com/catalystneuro/nwb-conversion-tools/issues/583